docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -204,57 +208,57 @@ Our default of using a NodePort service is the recommended way to expose the API
 
 .. tab-set::
 
-    .. tab-item:: NodePort (Default)
-        :sync: nodeport-tab
-
-        1. Make sure ports 30050 and 30051 are open on your nodes.
+    .. tab-item:: LoadBalancer (Default)
```

Comment:
Should we also change the "A Kubernetes cluster with ports 30050 and 30051 available for NodePort services" prerequisite at L15 to "A Kubernetes cluster with LoadBalancer or NodePort service support"?